src/platform_impl/macos/app_state.rs

```diff
@@ -40,10 +30,18 @@ use crate::{
     },
     window::WindowId,
 };
+use cocoa::{
+    appkit::{NSApp, NSApplication, NSWindow},
+    base::{id, nil},
+    foundation::NSSize,
+};
+use objc::{
+    rc::autoreleasepool,
+    runtime::{Object, BOOL, NO, YES},
+};
+use once_cell::sync::Lazy;
```

Comment:
Why did you move this below the `crate`-level imports?

Reply:
I ran `cargo fmt --all`, which I think automatically did this. I'll go ahead and undo this.